gen_errata.rb

```diff
@@ -367,44 +367,50 @@ def gen_ubuntu_errata(usn_db, packages, packages_by_name, release_whitelist=nil,
           next if release_whitelist.is_a?(Array) && !release_whitelist.include?(rel)
 
           # ESM-Updates do not have 'archs' -> Fallback to alternative handling
-          if dat.key?('archs')
+          if dat.key?('archs') && !packages.empty?
             add_packages_ubuntu(erratum, rel, dat, architecture_whitelist, packages)
           elsif dat.key?('sources')
             # try to do it the debian-way: get_binary_packages_for_erratum_package()
             warn "#{name} has no architectures for release #{rel} -> using source-packages as fallback" if @verbose
             dat['sources'].each do |source_pkg, pkg|
+              source_packages = [source_pkg]
+              # make sure we use signed linux-images instead of unsigned ones!
+              source_packages.unshift source_pkg.sub(/^linux/, 'linux-signed') if source_pkg =~ /^linux/
```

Comment:
If we want to use the signed instead of the unsigned, why do we keep the unsigned here?

Reply:
I am never fully sure if there are also supplementary packages that we want/need, but are only built from 'unsigned'.
Might be something to further investigate :sweat_smile: 

Given the amount of packages this accumulates to on Ubuntu, it is a valid idea to drop unsigned completely.